js/index.js

```diff
@@ -37,6 +37,76 @@ const siteContent = {
   },
 };
 
-// Example: Update the img src for the logo
-let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+let logoElem = document.getElementById("logo-img");
+logoElem.setAttribute('src', siteContent["nav"]["img-src"]);
+
+let ctaImgElem = document.querySelector("#cta-img");
+ctaImgElem.setAttribute('src', siteContent["cta"]["img-src"]);
+
+let navElem = document.querySelectorAll("header nav a");
+
+let navArray = Object.values(siteContent.nav);
+
+let i = 0;
+
+navElem.forEach(elem=>{
+  elem.textContent = navArray[i];
+  i++;
+})
```

Comment:
.forEach has 2 optional parameters: index and original array
```
navElem.forEach((el, i) => el.textContent = navArray[i]);
```
.map works the same way too